src/components/ContentHighlights/DeleteArchivedHighlightsDialogs.jsx

```diff
@@ -95,11 +95,10 @@ const DeleteArchivedCoursesDialogs = ({
           <p>
             <FormattedMessage
               id="highlights.highlights.tab.archived.courses.modal.message"
-              defaultMessage="Deleting the archived courses in this highlight will remove it from your learners{apostrophe} {doubleQoute}Find a Course{doubleQoute}. This action is permanent and cannot be undone."
+              defaultMessage="Deleting the archived courses in this highlight will remove it from your learners' {doubleQuote}Find a Course{doubleQuote}. This action is permanent and cannot be undone."
               description="Message shown in the modal to delete archived courses."
               values={{
-                apostrophe: "'",
-                doubleQoute: '"',
+                doubleQuote: '"',
```

Comment:
nit: this could use the same treatment as `apostrophe`; it's not really a dynamic property. Could the `"` be implemented like:

```jsx
"Deleting the archived courses in this highlight will remove it from your learners' \"Find a Course\". This action is permanent and cannot be undone."
```